samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/InteropRequests.kt

```diff
@@ -178,8 +179,10 @@ fun writeExternalStateToVault(
             it.map { linearId ->
                 println("Verification was successful and external-state was stored with linearId $linearId.\n")
 
-		val (valueByteArray, externalStateProof) = proxy.startFlow(::GetExternalStateByLinearId, linearId.toString()).returnValue.get()
-		val value = valueByteArray.toString(Charsets.UTF_8)
+		val response = proxy.startFlow(::GetExternalStateByLinearId, linearId.toString()).returnValue.get()
+		val responseString = response.toString(Charsets.UTF_8)
+        var responseJSON = JSONObject(responseString)
```

Comment:
Indentation of this line and the one below need to be fixed. They are not aligned with the lines above.